docs/src/developer/standards.rst

```diff
@@ -1,47 +1,86 @@
-.. contents:: Developer's Guide 103: Contribution Standards
+***********
+Conventions
+***********
 
-**********
-Contribute
-**********
+In this chapter, we present the different standards and guidelines we use throughout the project.
+All the conventions are enforced automatically during the PR process.
 
-Coding and Repository Standards
-===============================
-
-We are using flake8_ (along with some of its plugins) and pylint_.
-Their styles are provided in ``/tox.ini`` and ``/.pylintrc`` respectively.
+You can verify if your code will pass the checks locally beforehand using:
 
 .. code-block:: sh
 
-   tox -e flake8
-   tox -e pylint
+   tox -e lint
 
-Also, we are using a check-manifest_ which compares ``/MANIFEST.in`` and git
-structure of the source repository, and finally readme_renderer_ which
-checks whether ``/README.rst`` can be
-actually rendered in PyPI_ website page for Oríon.
+.. _standard-coding:
 
-.. code-block:: sh
+Coding standard
+===============
 
-   tox -e packaging
+Our coding standards are specified via flake8_ and pylint_. Their configurations are provided in
+``tox.ini`` and ``.pylintrc`` respectively. You can verify the conformity of your changes locally
+by running ``$ tox -e flake8`` and ``$ tox -e pylint``.
```

Comment:
I would add a note about docstring standards here and point to numpy's documentation: https://numpydoc.readthedocs.io/en/latest/format.html